<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reminder_Window</class>
 <widget class="QWidget" name="Reminder_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reminder</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255);
	font: 11pt &quot;Roboto&quot;;
}</string>
  </property>
  <widget class="QCommandLinkButton" name="less_options_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton
{
	font: 11pt &quot;Roboto&quot;;
	text-decoration: underline;
	
	color: rgb(68, 137, 206);
	qproperty-icon : None
}</string>
   </property>
   <property name="text">
    <string>Less Options?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
color: #fff;
	font: 11pt &quot;Roboto&quot;;
border-style: solid;
border-color: rgb(79, 159, 239);
border-width: 1px;
border-radius: 4px;
	
	background-color: rgb(79, 159, 239);
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
color:#fff;
	font: 11pt &quot;Roboto&quot;;
border-style: solid;
border-color: rgb(212, 0, 0);
border-width: 1px;
border-radius: 4px;
	background-color: rgb(212, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	font: 11pt &quot;Roboto&quot;;
}</string>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="repetition_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	font: 11pt &quot;Roboto&quot;;
}</string>
   </property>
   <property name="text">
    <string>Repetition</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="event_title_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	font:  11pt &quot;Roboto&quot;;
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 4px;
	background-color: rgb(255, 255, 255)
}</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>                               Remind me about.....</string>
   </property>
  </widget>
  <widget class="QLabel" name="reminder_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	font: 11pt &quot;Roboto&quot;;
}</string>
   </property>
   <property name="text">
    <string>Remind Me</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="target_date_edit">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit
{
	font: 11pt &quot;Roboto&quot;;
}</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTimeEdit" name="target_time_edit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>118</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit
{
	font: 11pt &quot;Roboto&quot;;
}</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="repetition_options">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAlphabetically</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>One time event</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Daily</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Weekly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monthly</string>
    </property>
   </item>
  </widget>
  <widget class="QCommandLinkButton" name="more_options_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton
{
	font: 11pt &quot;Roboto&quot;;
	text-decoration: underline;
	
	color: rgb(68, 137, 206);
	qproperty-icon : None
}</string>
   </property>
   <property name="text">
    <string>More Options?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>68</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	font: 11pt &quot;Roboto&quot;;
}</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="reminder_options">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>On time</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 mins early</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10 mins early</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 hour early</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 hours early</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 day early</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
